ohapi/api.py

```diff
@@ -116,38 +119,26 @@ def exchange_oauth2_member(access_token):
 def upload_file(target_filepath, metadata, access_token, base_url=OH_BASE_URL,
```

Comment:
I noticed that this function now does not do any error handling in case any of the upload-steps break. Could you (analogously to #30) add this to make sure potential errors are caught?

See https://github.com/gedankenstuecke/oh_data_uploader/blob/master/main/views.py#L61 for an example for how it could look like for the AWS upload method.

Reply:
Yeah I had added those but removed it just a day ago as I thought that `vcrpy` would be enough. nevertheless, I will add them back. Thank you!